.travis.yml

```diff
@@ -25,6 +25,8 @@ services:
   - redis-server
 
 install:
+  - go get github.com/TykTechnologies/dq
+  - go get github.com/TykTechnologies/tyk-cluster-framework/...
```

Comment:
This will be vendored when that library is stable, they are currently codependent, one informs the utility of the other somewhat, for now I didn't want to introduce any vendoring until I'm happy with the lib across Gateway, Dashboard and Pump